src/main/java/org/jboss/xavier/analytics/pojo/output/workload/summary/ComplexityModel.java

```diff
@@ -0,0 +1,120 @@
+package org.jboss.xavier.analytics.pojo.output.workload.summary;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import javax.persistence.*;
+
+@SqlResultSetMapping(
+        name = "mappingComplexityModels",
+        classes = @ConstructorResult(
+                targetClass = ComplexityModel.class,
+                columns = {
+                        @ColumnResult(name = "easy", type = Integer.class),
+                        @ColumnResult(name = "medium", type = Integer.class),
+                        @ColumnResult(name = "difficult", type = Integer.class),
+                        @ColumnResult(name = "unknown", type = Integer.class)
+                }
+        )
+)
+
+@NamedNativeQuery(
+        name = "ComplexityModel.calculateComplexityModels",
+        query = "select sum(case when lower(complexity)='easy' then 1 else 0 end) as easy, sum(case when lower(complexity)='medium' then 1 else 0 end) as medium, sum(case when lower(complexity)='difficult' then 1 else 0 end) as difficult, sum(case when (complexity is null or lower(complexity)='unknown') then 1 else 0 end) as \"unknown\" from workload_inventory_report_model where analysis_id = :analysisId",
```

Comment:
```suggestion
        query = "select sum(case when lower(complexity)='easy' then 1 else 0 end) as easy, sum(case when lower(complexity)='medium' then 1 else 0 end) as medium, sum(case when lower(complexity)='hard' then 1 else 0 end) as hard, sum(case when (complexity is null or lower(complexity)='unknown') then 1 else 0 end) as \"unknown\" from workload_inventory_report_model where analysis_id = :analysisId",
```

Reply:
Thanks for discovering this @mrizzi , I've already changed "difficult" by "hard" not just in the query but in the Class field.